programs/github

```diff
@@ -144,10 +147,4 @@ if action == 'auth' then
 	return
 end
 
-if not validActions[action] then
-	for k, v in pairs(validActions) do
-		print(v.usage)
-	end
-else
-	print(validActions[action].usage)
-end
+print(action.usage)
```

Comment:
This is handled above now... We shouldn't get here unless action is defined and valid, but missused for some reason.